vck/src/commonMain/kotlin/at/asitplus/wallet/lib/agent/Issuer.kt

```diff
@@ -12,7 +16,7 @@ import kotlinx.datetime.Instant
  *
  * It can issue Verifiable Credentials, revoke credentials and build a revocation list.
  */
-interface Issuer {
+interface Issuer : ReferencedTokenIssuer<CredentialToBeIssued, KmmResult<Issuer.IssuedCredential>>, StatusIssuer<String, ByteArray>, StatusProvider<Any> {
```

Comment:
Why is the generic for `StatusProvider` only `Any` and nothing more specific?

Reply:
fixed in 93ce860696aa4077acb9a053a8b3eac9bee2733b